unity/EditorPlugin/PluginEntryPoint.cs

```diff
@@ -544,7 +544,7 @@ private static void InitEditorLogPath(EditorPluginModel editorPluginModel)
       editorPluginModel.PlayerLogPath.SetValue(playerLogPath);
     }
 
-    internal static readonly string LogPath = Path.Combine(Path.Combine(Path.GetTempPath(), "Unity3dRider"), $"{DateTime.Now:yyyy-MM-ddT-HH-mm-ss}.log");
```

Comment:
`DateTime.UtcNow` might help if you still want to use date. `Now` is slow because it has to look up timezone information.

Reply:
I tried that. It was still slow